### DISCUSSION TITLE
Add dark theme to https://react-icons.github.io/

### DISCUSSION THREAD

**Discussion by K3RIoT** [react-icons/react-icons#691] [Ideas]
Hello, thank you for the package you provided
I have an idea, it would be good if for the website
Also add a dark theme, because personally, it hurts my eyes at night when I'm searching for an icon

**Comment by joshistoast** [comment]
My eyes would be blessed by this 👀 🙌

**Comment by serter1** [comment]
👍🏻👍🏻
